pkg/controller/migmigration/task.go

```diff
@@ -116,6 +117,7 @@ var CancelItinerary = Itinerary{
 	{phase: Canceling},
 	{phase: EnsureStagePodsDeleted, flags: HasStagePods},
 	{phase: EnsureAnnotationsDeleted, flags: HasPVs},
+	{phase: BackupDeletionRequested},
```

Comment:
Shouldn't the restores get deleted on the destination when a failed migration is canceled?